### ISSUE TITLE
Class Shopmarks Not Found

### ISSUE THREAD

**Issue by XedinUnknown** [vendidero/woocommerce-germanized#122]
Hi!

I install your plugin using WP CLI in a Docker container. Upon activation, I get the following:

```
Activating 'woocommerce-germanized'...
[14-Dec-2020 16:48:11 UTC] PHP Fatal error:  Uncaught Error: Class 'Vendidero\Germanized\Shopmarks' not found in /var/www/html/wp-content/plugins/woocommerce-germanized/includes/wc-gzd-core-functions.php:42
Stack trace:
#0 /var/www/html/wp-content/plugins/woocommerce-germanized/includes/wc-gzd-template-hooks.php(27): wc_gzd_get_single_product_shopmarks()
#1 /var/www/html/wp-content/plugins/woocommerce-germanized/woocommerce-germanized.php(625): include_once('/var/www/html/w...')
#2 /var/www/html/wp-content/plugins/woocommerce-germanized/woocommerce-germanized.php(508): WooCommerce_Germanized->frontend_includes()
#3 /var/www/html/wp-content/plugins/woocommerce-germanized/woocommerce-germanized.php(186): WooCommerce_Germanized->includes()
#4 /var/www/html/wp-content/plugins/woocommerce-germanized/woocommerce-germanized.php(113): WooCommerce_Germanized->__construct()
#5 /var/www/html/wp-content/plugins/woocommerce-germanized/woocommerce-germanized.php(1162): WooCommerce_Germanized::instance()
#6 /var/www/html/wp-content/plugins/woocommerce-germanized/wooco in /var/www/html/wp-content/plugins/woocommerce-germanized/includes/wc-gzd-core-functions.php on line 42
```

Any ideas?

**Comment by vendidero** [comment]
Hi there,

yes, that's the same issue I'm currently facing when deploying our (pro) tests. Seems like an issue related to the Jetpack Autoloader we are using (> 2.6.0). I'll guess I will downgrade to 2.6.0 upon next release.

**Comment by XedinUnknown** [comment]
Ok, wonderful! Thanks for the quick response!

**Comment by XedinUnknown** [comment]
Could you give a clue about when the next fixed version will be released, please?

**Comment by vendidero** [comment]
Fixed within 3.3.3
